### PR TITLE
Methods naming change & FormBuilder extension

### PR DESCRIPTION
**Pull request by bear-in-mind**
### Switched method names
- In model, `abyme_for` became `abymize`
- In view, we now use `abyme_for`

### FormBuilder extension
- We can now use either `f.abyme_for(:association,  options)` or like before, `abyme_for(:association, f, options)`
- In the `builder.rb` file, we successfully extended Rails `FormBuilder` class with an `abyme_for` method that call our view helper on the `@template` object (which is the `ActionView` instance generated by the `FormBuilder`)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,29 @@
...
+# views/projects/_form.html.erb
+<%= simple_form_for @project do |f| %>
+  <%= f.input :title %>
+  <%= f.input :description %>
+  <%= f.submit 'Save' %>
+
+  <%= f.abyme_for(:tasks) %>
+<% end %>
+```
+Supposing you have a partial located in `views/abyme/_task_fields` containing your fields for `tasks`, this command will generate and display 3 elements in this order :
```
Comment (charlottepages) [inline README.md:19]: unclear which command you are referring to
```suggestion
Supposing you have a partial located in `views/abyme/_task_fields` containing your fields for `tasks`, the `abyme_for` command will generate and display 3 elements in this order :
```
?

File: README.md [hunk 2]
```diff
@@ -142,15 +159,15 @@ abyme provides helper methods for all these. Here's how our form for `Project` l
...
+  <%= f.abyme_for(:tasks) do |abyme| %>
     <%= abyme.records %>
     <%= abyme.new_records %>
     <%= add_association %>
   <% end %>
 <% end %>
 ```
 
-`abyme.records` will contain the persisted associations fields, while `abyme.new_records` will contain fields for the new associations. `add_association` will by default generate a button with a text of type "Add `resource_name`". To work properly, this method **has** to be called **inside the block** passed to the `abymize` method.
+`abyme.records` will contain the persisted associations fields, while `abyme.new_records` will contain fields for the new associations. `add_association` will by default generate a button with a text of type "Add `resource_name`". To work properly, this method **has** to be called **inside the block** passed to the `abyme_for` method.
```
Comment (charlottepages) [inline README.md:170]: sorry wording again ^^ : unsure about `associations` 
which imo refers to the connection between two models 
wb: `associated records` or `child records` ?
👉 idem for `add_association`

File: README.md [hunk 3]
```diff
@@ -202,31 +219,31 @@ Which is the way you would configure `nested_attributes` 90% of the time. Should
...
 ```ruby
-  <%= abymize(:tasks, f) do |abyme| %>
+  <%= f.abyme_for(:tasks) do |abyme| %>
     <%= abyme.records(order: { created_at: :asc }) %>
     <%= abyme.new_records %>
     <%= add_association %>
   <% end %>
 ```
-* `partial:` : allows you to indicate a custom partial, if one has not already been passed to `abymize`.
+* `partial:` : allows you to indicate a custom partial, if one has not already been passed to `abyme_for`.
```
Comment (charlottepages) [inline README.md:236]: can you also write this then: 
```
<%= f.abyme_for(:tasks, partial: 'projects/task_fields') do |abyme| %>
  <%= abyme.records(order: { created_at: :asc }) %>
  <%= abyme.new_records %>
  <%= add_association %>
<% end %>
```
?

**Review by charlottepages — APPROVED** [review]
nothing that cannot be handled in later PR 👍

File: README.md (hunk 2 above)
Comment (bear-in-mind) [reply README.md:170]: You're right ! We kept `cocoon`'s original wording without questioning it. Let's change it in the next PR !

File: README.md (hunk 3 above)
Comment (bear-in-mind) [reply README.md:236]: Roger, it's documented below for the `#abyme_for` part

**Pull request merged**